server/server_test.go

```diff
@@ -1713,8 +1713,10 @@ func generateJSONBenchmarkData(k, v int) map[string]interface{} {
 	}
 
 	return map[string]interface{}{
-		"keys":   keys,
-		"values": values,
+		"input": map[string]interface{}{
```

Comment:
This is a minor refactoring that silences "'input' key missing" warnings from the testcases.